### PR TITLE
[release-4.10] OCPBUGS-3968: set proper pod security ns labels

### PR DESCRIPTION
**Pull request by ardaguclu**
This is manual cherrypick of this https://github.com/openshift/oc/pull/1187

### CODE REVIEW

**Comment by ardaguclu** [comment]
/hold
until the pre-merge testing is completed.

**Review by soltysh — APPROVED** [review]
/approve

**Comment by zhouying7780** [comment]
pre-merge test  done

**Pull request merged**
